plugins-compat-tester/src/main/resources/resultToReport.xsl

```
@@ -275,9 +275,12 @@ th.version {
 	        </xsl:when>
-	        <xsl:otherwise>
+            <xsl:when test="$img='document.png'">
+                <xsl:attribute name="src">data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAQAAABKfvVzAAABNklEQVR4XpWRMU4zQQyFP89ONvmlUMIfKQUXoKfnCtBzAG7CBTgANYoUUdFyA46BRIcSkp0ZIyE0luUKr+xm/Om9583K3yoDwOPl7CYDjSONiUKjUig/c6Jw3N6/GsDF0/laSXg9tcnbLWcGMOZPCiMNQaC31XLmLCny+30EQDlFUBxAB+gT/FtUINmjQ5Kx0VKKGZBoqdHMEngNu54BsOefN+QUgqXKCRVibIMd0A2pqXhLNA8A3gxLlghYXA9oN7FCSF3FkHClHTMA3v06oKwioGQGFxPEZRC8AnOqrYLTEDRkcKvxnBL/QyxQ1ggNjZZM8j8J6a32igd6i7Utxyvt2hxlABfXEsBXc8Dz3cs1Mi3qoox1bFkHSC2VNOVjPoz7VA/bKwPgYcOm30YYGBAqVRu++Abw8GiSNi5W4wAAAABJRU5ErkJggg==</xsl:attribute>
+            </xsl:when>
+            <xsl:when test="$img='terminal.png'">
```

I removed the otherwise, as it can mask other errors as with this one.